expression/builtin_time_vec.go

```diff
@@ -803,11 +803,42 @@ func (b *builtinTimestampAddSig) vecEvalString(input *chunk.Chunk, result *chunk
 }
 
 func (b *builtinToDaysSig) vectorized() bool {
-	return false
+	return true
 }
 
+// evalInt evals a builtinToDaysSig.
+// See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_to-days
 func (b *builtinToDaysSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETDatetime, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalTime(b.ctx, input, buf); err != nil {
```

Comment:
Should we also check `handleInvalidTimeError` here?

Reply:
It's `args[0]`'s responsibility.